exp/lib/channel.py

```diff
@@ -122,5 +122,8 @@ def listen(self, name=None, callback=None):
       self._listeners[name] = []
     self._listeners[name].append(callback)
 
+  def fling(self, uuid=None):
+    return self.broadcast(name='fling', payload={ 'uuid': uuid })
```

Comment:
You love nitpicks right? The spaces between the curly braces are a PEP-8 violation.

```
def fling(self, uuid=None):
    return self.broadcast(name='fling', payload={'uuid': uuid})
```